aggregator/src/aggregator/http_handlers.rs

```diff
@@ -612,11 +595,33 @@ async fn hpke_config_cors_preflight(mut conn: Conn) -> Conn {
 async fn upload<C: Clock>(
     conn: &mut Conn,
     (State(aggregator), BodyBytes(body)): (State<Arc<Aggregator<C>>>, BodyBytes),
```

Comment:
We'll still read the entire body off the wire before we start processing the first report, right? I think that's fine just to reach -16 conformity and prove that the thing works, but we should at least note that more work is needed here to stream requests.

Reply:
That's right, `impl TryFromConn for BodyBytes` reads the whole body. I've filed https://github.com/divviup/janus/issues/4149 for this.